### PR TITLE
D2d 045

### PR DESCRIPTION
**Pull request by Nibaron**
verified.
WhatsApp Image 2024-01-25 at 22 24 07_2a973136

Not Verified
image

### CODE REVIEW

File: app/auth/verify-email/page.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import EmailVerificationModal from '@/components/email-verification-modal';
+import { useEffect, useState } from 'react';
+
+
+const VerifyEmailPage = () => {
+  const [showModal, setShowModal] = useState(true);
+
+  return (
+    <>
+    <div></div>
```
Comment (tajnur007) [inline app/auth/verify-email/page.tsx:12]: Remove this empty `div` and above fragment

File: assets/icons.tsx [hunk 2]
```diff
@@ -780,3 +780,79 @@ export const DeleteIcon = () => {
...
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'>
+      <rect
+        x='10'
+        y='10.5'
+        width='56'
+        height='56'
+        rx='28'
+        fill='#FF0000'
+        fill-opacity='0.24'
```
Comment (tajnur007) [inline assets/icons.tsx:837]: Re-write this kind of attributes `fillOpacity` this way.

File: components/email-verification-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import Modal from 'react-modal';
+import { ArrowLeftIcon, CheckCircleIcon, CrossCircleIcon } from '@/assets/icons';
+import { Button } from './button';
+
+// if (Modal.defaultStyles.overlay) {
```
Comment (tajnur007) [inline components/email-verification-modal.tsx:5]: If this commented code is no longer required then remove this code.

**Comment by Nibaron** [comment]
All issue fixed.
image

**Review by tajnur007 — APPROVED** [review]
PR approved!

**Pull request merged**